packages/server/customer-os-api/service/organization_service.go

```diff
@@ -470,7 +471,21 @@ func (s *organizationService) ReplaceOwner(ctx context.Context, organizationID,
 	tracing.SetDefaultServiceSpanTags(ctx, span)
 	span.LogFields(log.String("organizationID", organizationID), log.String("userID", userID))
 
-	dbNode, err := s.repositories.OrganizationRepository.ReplaceOwner(ctx, common.GetTenantFromContext(ctx), organizationID, userID)
+	// dbNode, err := s.repositories.OrganizationRepository.ReplaceOwner(ctx, common.GetTenantFromContext(ctx), organizationID, userID)
```

Comment:
remove this comment
also remove Replace Owner implementation in repository if not used anymore